test-app/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
IMO, we should have 2 set's of examples, one will be app router and other will be page router

Reply:
Added page-router example